agentlocal/agentlocal.go

```diff
@@ -69,8 +74,14 @@ type AgentStatus struct {
 
 // GetStatus returns local pmm-agent status.
 // As a special case, if pmm-agent is running, but not set up, ErrNotSetUp is returned.
-func GetStatus() (*Status, error) {
-	res, err := agentlocal.Default.AgentLocal.Status(nil)
+func GetStatus(getNetworkInfo bool) (*Status, error) {
```

Comment:
In many cases, it is better to use "enum"-like constants instead of boolean parameters. Compare `GetStatus(PingServer)` and `GetStatus(true)`.